.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: make
+    command: ./maze
```

Comment:
Makefile is enough and portable. I don't think I like to include this.

Reply:
Not here to convince you. A Makefile and Gitpod serve two different purposes: with the `.gitpod.yml` everyone can try maze without needing to install a thing on their machine.  And it uses make :) 

That said, feel free to use the code in this PR as you see fit.